### ISSUE TITLE
Bug: Cannot build stt_ios.framework

### ISSUE THREAD

**Issue by normano** [coqui-ai/STT#2090]
**Describe the bug**

I was trying to generate the stt_ios.framework on my M1 Mac.
I've checked out the coqui stt report and switched to the tf2.8 branch and got to the point of generating a libstt.so.
Now I'm trying to use bazel to build the stt_ios.framework file to use in an iOS project.

ERROR: file 'native_client/libkenlm.so' is generated by these conflicting actions:
Label: //native_client:libkenlm.so, //native_client:kenlm
RuleClass: cc_binary rule, cc_library rule
Configuration: ec5d3b85eb62bfce4f44b561d3ea30a2f9ab7f1fef850bc156a48de4d3653a88
Mnemonic: CppLink, Fail
Action key: 006947bcf13de816e44b1a341b2c67215cb34e347186773366c029f692e4af82, 11e223d1a51b8d11d33659b7fd7a03ba0c0fb8bdffd78d36bb696454635d2200
Progress message: Linking native_client/libkenlm.so, Reporting failed target //native_client:kenlm located at /Users/norm/projects/speech_rek/STT/tensorflow/native_client/BUILD:110:11
PrimaryInput: File:[[<execution_root>]bazel-out/ios-arm64-min9.0-applebin_ios-ios_arm64-opt-ST-c2e45b3824d7/bin]native_client/_objs/libkenlm.so/bhiksha.o, (null)
PrimaryOutput: File:[[<execution_root>]bazel-out/ios-arm64-min9.0-applebin_ios-ios_arm64-opt-ST-c2e45b3824d7/bin]native_client/libkenlm.so
ERROR: com.google.devtools.build.lib.skyframe.ArtifactConflictFinder$ConflictException: com.google.devtools.build.lib.actions.MutableActionGraph$ActionConflictException: for native_client/libkenlm.so, previous action: action 'Reporting failed target //native_client:kenlm located at /Users/norm/projects/speech_rek/STT/tensorflow/native_client/BUILD:110:11', attempted action: action 'Linking native_client/libkenlm.so'

**Expected behavior**

Expected a .framework file to appear magically for use in an ios project.

**Environment (please complete the following information):**
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: MacOS 12.2
- **TensorFlow installed from (our builds, or upstream TensorFlow)**: your build
- **TensorFlow version (use command below)**: 2.8
- **Python version**: 3.9
- **Bazel version (if compiling from source)**: 4.2.1
- **GCC/Compiler version (if compiling from source)**: Apple clang version 13.0.0 (clang-1300.0.29.30), Target: arm64-apple-darwin21.3.0
- **CUDA/cuDNN version**: N/A
- **GPU model and memory**: Apple M1
- **Exact command to reproduce**: BAZEL_ENV_FLAGS="TF_NEED_CUDA=0" /opt/homebrew/Cellar/bazel@4.2.1/4.2.1_1/bin/bazel build --workspace_status_command="bash native_client/bazel_workspace_status_cmd.sh" -c opt --config=ios_arm64 --define=runtime=tflite --copt=-DTFLITE_WITH_RUY_GEMV //native_client:stt_ios

**Additional context**

Would be better to push out a framework file already built into a github release like the other files. Not sure though how others have made ios apps that use this library.

**Comment by reuben** [comment]
I've fixed the build for the STT framework and added a framework build for KenLM. I've also started building the Swift bindings as a Swift module in CI, and it's working well, but I haven't been able to figure out how to put all three components together: either embedding the Swift bindings in the STT framework, or referencing the module directly somehow. Any feedback appreciated! Pre-built binaries can be grabbed from the CI artifacts, the package is called `ios_artifacts.zip`.
